sonar-db/src/main/java/org/sonar/db/purge/PurgeConfiguration.java

```diff
@@ -63,6 +62,10 @@ public IdUuidPair rootProjectIdUuid() {
     return scopesWithoutHistoricalData;
   }
 
+  public Collection<String> getDisabledComponentUuids() {
+    return disabledComponentUuids;
+  }
+
   @CheckForNull
   public Date maxLiveDateOfClosedIssues() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)